src/App.js

```diff
@@ -2,44 +2,46 @@ import "./App.css";
 import Tweet from "./components/Tweet";
 
 const tweetsArray = [
-  {
-    user: {
-      name: "Thoughts of Dog®",
-      image: "https://i.imgur.com/b0EdHVV.jpg",
-      handle: "dog_feelings",
-    },
-    timestamp: "1h ago",
-    message:
-      "the human likes to say. that i live here rent free. but i would argue. this housing accommodation. is my payment. for a lifetime of love. and excellent company",
-  },
-  {
-    user: {
-      name: "Thoughts of Dog®",
-      image: "https://i.imgur.com/b0EdHVV.jpg",
-      handle: "dog_feelings",
-    },
-    timestamp: "2h ago",
-    message:
-      "sometimes. the human presses their noggin against mine. to figure out what i’m thinking. so i just think really hard. about how much i love them. and hope they figure it out",
-  },
-  {
-    user: {
-      name: "Thoughts of Dog®",
-      image: "https://i.imgur.com/b0EdHVV.jpg",
-      handle: "dog_feelings",
-    },
-    timestamp: "3h ago",
-    message:
-      "here is what. i plan to accomplish today: \n\n2. bark loudly. but at nothing \n7. lose my ball under the couch\n7b. politely ask the human. to get my ball\n3. immediately lose it again. under the same couch\n4. big nap. you have worked hard\n2. repeat",
-  },
+	{
+		user: {
+			name: "Thoughts of Dog®",
+			image: "https://i.imgur.com/b0EdHVV.jpg",
+			handle: "dog_feelings",
+		},
+		timestamp: "1h ago",
+		message:
+			"the human likes to say. that i live here rent free. but i would argue. this housing accommodation. is my payment. for a lifetime of love. and excellent company",
+	},
+	{
+		user: {
+			name: "Thoughts of Dog®",
+			image: "https://i.imgur.com/b0EdHVV.jpg",
+			handle: "dog_feelings",
+		},
+		timestamp: "2h ago",
+		message:
+			"sometimes. the human presses their noggin against mine. to figure out what i'm thinking. so i just think really hard. about how much i love them. and hope they figure it out",
+	},
+	{
+		user: {
+			name: "Thoughts of Dog®",
+			image: "https://i.imgur.com/b0EdHVV.jpg",
+			handle: "dog_feelings",
+		},
+		timestamp: "3h ago",
+		message:
+			"here is what. i plan to accomplish today: \n\n2. bark loudly. but at nothing \n7. lose my ball under the couch\n7b. politely ask the human. to get my ball\n3. immediately lose it again. under the same couch\n4. big nap. you have worked hard\n2. repeat",
+	},
 ];
 
 function App() {
-  return (
-    <div className="App">
-      <Tweet />
-    </div>
-  );
+	return (
+		<div className="App">
+			<Tweet tweet={tweetsArray[0]}></Tweet>
+			<Tweet tweet={tweetsArray[1]}></Tweet>
+			<Tweet tweet={tweetsArray[2]}></Tweet>
```

Comment:
Here, you should've used a loop to render multiple components at once, like this for example:
```
{tweetsArray.map((element) => {
       return (
           <Tweet tweet={element} />
       )
})}
```